lms/djangoapps/program_enrollments/models.py

```diff
@@ -39,12 +39,12 @@ class ProgramEnrollment(TimeStampedModel):  # pylint: disable=model-missing-unic
 
     class Meta(object):
         app_label = "program_enrollments"
-        unique_together = ('external_user_key', 'program_uuid', 'curriculum_uuid')
```

Comment:
As a side note, it's funny that we declared and then immediately redefined this before.